data/data/bootstrap/bootstrap-in-place/files/opt/openshift/bootstrap-in-place/bootstrap-in-place-post-reboot.sh

```diff
@@ -24,41 +24,73 @@ function signal_bootstrap_complete {
   done
 }
 
+function expedite_bootstrapping {
```

Comment:
lets break it down in several commits? That would allow us reverting specific bits if it gets fixed upstream